tree/ntuple/v7/test/CustomStructLinkDef.h

```diff
@@ -16,4 +16,7 @@
 
 #pragma link C++ class ComplexStruct+;
 
+#pragma link C++ class BaseOfStructWithEnums + ;
+#pragma link C++ class StructWithEnums + ;
```

Comment:
```suggestion
#pragma link C++ class BaseOfStructWithEnums+;
#pragma link C++ class StructWithEnums+;
```

Reply:
As discussed in the Shush channel, that's `clang-format`s decision

Reply:
clang-format should probably not be run on LinkDef files.

Reply:
All right, I created #10448 to keep track of it.

Reply:
I guess `rootcling` parses this correctly even with the extra whitespace?